src/App.jsx

```diff
@@ -1,104 +1,28 @@
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
 
-export const App = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
-
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Sincere@april.biz">
-              Leanne Graham
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae ea
-          dolores neque fugiat blanditiis voluptate porro vel nihil molestiae ut
-          reiciendis qui aperiam non debitis possimus qui neque nisi nulla
-        </p>
-
-        <hr />
-
-        <b data-cy="NoCommentsMessage">No comments yet</b>
-      </div>
-
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">doloremque illum aliquid sunt</h3>
+function findUser(userId) {
+  return usersFromServer.find(el => el.id === userId);
+}
 
-          <p>
-            {' Posted by  '}
+function findComments(id) {
+  return commentsFromServer.filter(el => el.postId === id);
+}
 
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
+export const posts = postsFromServer.map(post => ({
+  ...post,
+  user: findUser(post.userId),
+  comments: findComments(post.id),
+}));
```

Comment:
Ensure that the `postsFromServer`, `usersFromServer`, and `commentsFromServer` JSON files contain the expected structure. If any of these files have unexpected data formats, it could lead to runtime errors when accessing properties like `userId` or `postId`.